### DISCUSSION TITLE
Replacing some pages with dialogs

### DISCUSSION THREAD

**Discussion by ChristopherDKeller** [remsfal/remsfal-frontend#295] [General]
In order to improve UX/UI, I would suggest using dialogs from primevue (https://primevue.org/dialog/) in place of some pages. An example where this could be useful are the pages used to create a project or an "Eigentum". These could be remodeled into dialogs, which in my opinion would improve the clarity of the workflow. What are other pages that could be replaced?

**Comment by TheoThmn** [comment]
This touches our discussion on #285
